README.md

```diff
@@ -8,15 +8,30 @@
 # aries-framework-go
 
 ## Introduction
-A go framework for Aries
+A Go framework for Aries
 
-## Contributing
+## Build
+##### Prerequisites (General)
+- Go 1.12
 
-Found a bug? Ready to submit a PR? Want to submit a proposal for your grand
-idea? Follow our [guidelines](.github/CONTRIBUTING.md) for more information
-to get you started!
+##### Prerequisites (for running tests)
+- Go 1.12
+- Docker
+- Make
+
+##### Targets
+```
+# run all the project build targets
+make all
+
+# run linter checks
+make checks
+
+# run unit tests
+make unit-test
+```
 
-### Crypto Material generation for tests
+##### Crypto Material generation for tests
```

Comment:
inconsistent capitalization (Crypto **M**aterial)

Reply:
didn't notice this as it was an existing line. [Updated](https://github.com/hyperledger/aries-framework-go/compare/1c758ac9c8a5372da72239e742bd1fd802b7b515..45fafadc2e71b354f413c11155c1c1f6f9212c03#diff-04c6e90faac2675aa89e2176d2eec7d8R34) now.